auto_submit/lib/requests/check_pull_request.dart

```diff
@@ -27,19 +27,29 @@ class CheckPullRequest extends RequestHandler {
 
   final PubSub pubsub;
 
-  Future<Response> get() async {
-    final pub.PullResponse pullResponse = await pubsub.pull(1, 'auto-submit-queue-sub');
+  Future<List<Response>> get() async {
+    List<Response> responses = <Response>[];
+    final pub.PullResponse pullResponse = await pubsub.pull('auto-submit-queue-sub', 100);
     final List<pub.ReceivedMessage>? receivedMessages = pullResponse.receivedMessages;
     if (receivedMessages == null) {
       log.info('There are no more messages available in the backlog');
-      return Response.ok('No more pull requests in the pubsub.');
+      responses.add(Response.ok('No more pull requests in the pubsub.'));
```

Comment:
nit: Since this isn't processing, we shouldn't say "more." Exiting with "No requests in the queue" would be the correct terminology

Reply:
Gotcha, changed the log info and response message to make it clear.